clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2661,7 +2661,7 @@ def VTTAddrPointOp : CIR_Op<"vtt.address_point",
   let arguments = (ins OptionalAttr<FlatSymbolRefAttr>:$name,
                        Optional<CIR_AnyType>:$sym_addr,
                        I32Attr:$offset);
-  let results = (outs Res<CIR_PointerType, "", []>:$addr);
+  let results = (outs CIR_VoidPtrType:$addr);
```

Comment:
Does the `Res<>` wrapper in the original code here do anything or does the fact that the description and decorators list are both empty make it redundant?

Reply:
yes, `Res` wrapper with empty description and decorator list does nothing here